axelor-business-project/src/main/resources/views/ExpenseLine.xml

```diff
@@ -18,6 +18,20 @@
         <field name="expense.statusSelect" hidden="true"/>
     </grid>
     
+    <grid name="expense-line-invoicing-project-grid" title="Expense lines" model="com.axelor.apps.hr.db.ExpenseLine" orderBy="-expenseDate" groupBy="project">
+		<hilite if="untaxedAmount == 0" color="danger"/>
+		<field name="project" requiredIf="toInvoice" canView="false" widget="SuggestBox" width="150"/>
+		<field name="expenseProduct" canView="false" widget="SuggestBox" form-view="product-expense-form" grid-view="product-expense-grid" width="200"/>
+		<field name="expenseDate" width="120"/>
+		<field name="user" hidden="true" form-view="user-form" grid-view="user-grid"/>
+		<field name="totalAmount" aggregate="sum" width="170"/>
+		<field name="totalTax" aggregate="sum" readonlyIf="expenseProduct.blockExpenseTax" width="80"/>
+        <field name="toInvoice"/>
+		<field name="comments" width="400"/>
+		<field name="untaxedAmount" hidden="true"/>
+		<field name="expenseProduct.blockExpenseTax" hidden="true"/>
```

Comment:
you can remove attributes like requiredIf, canView, widget etc. as it is not useful for this new grid view.